wandb/integration/openai/fine_tuning.py

```diff
@@ -5,15 +5,25 @@
 import time
 from typing import Any, Dict, Optional, Tuple
 
-import openai
+from pkg_resources import parse_version
 
 import wandb
 from wandb import util
 from wandb.data_types import Table
 from wandb.sdk.lib import telemetry
 from wandb.sdk.wandb_run import Run
 
-openai_version = openai.__version__
+openai = util.get_module(
+    name="openai",
+    required="Error: `openai` not installed >> This integration requires openai!  To fix, please `pip install openai`",
```

Comment:
```suggestion
    required="`openai` not installed >> This integration requires openai!  To fix, please `pip install openai`",
```

no need for the error it will raise wandb.Error by default